tests/failing-symbolic.haskell-booster-dev

```diff
@@ -135,4 +135,8 @@ tests/specs/mcd-structured/flopper-tick-pass-rough-spec.k
 tests/specs/mcd-structured/functional-spec.k
 tests/specs/mcd-structured/pot-join-pass-rough-spec.k
 tests/specs/mcd-structured/vow-fess-fail-rough-spec.k
-tests/specs/mcd-structured/vow-flog-fail-rough-spec.k
\ No newline at end of file
+tests/specs/mcd-structured/vow-flog-fail-rough-spec.k
+tests/specs/mcd/dai-adduu-fail-rough-spec.k
+tests/specs/mcd/flipper-addu48u48-fail-rough-spec.k
+tests/specs/mcd-structured/dai-adduu-fail-rough-spec.k
+tests/specs/mcd-structured/flipper-addu48u48-fail-rough-spec.k
```

Comment:
Well that's less than ideal I guess.....

Reply:
It is, but the booster will adapt and be more expressive, as per [this issue](https://github.com/runtimeverification/haskell-backend/issues/4053).

Reply:
This might allow us to prove further tests using just `booster-dev`.